tests/nemo_text_processing/hy/data_inverse_text_normalization/test_cases_word.txt

```diff
@@ -0,0 +1,50 @@
+~
```

Comment:
if here is needed to write all latin letters in armenian let me know, for example aabach~աաբասհ

Reply:
There is no need to write latin letters in armenian.